SPECS-EXTENDED/osinfo-db/osinfo-db.spec

```diff
@@ -31,10 +69,73 @@ osinfo-db-import --root %{buildroot} --dir %{_datadir}/osinfo %{SOURCE0}
 %{_datadir}/osinfo/install-script
 %{_datadir}/osinfo/schema
 
+%if %{with_mingw}
+%files -n mingw32-osinfo-db
+%dir %{mingw32_datadir}/osinfo
+%doc %{mingw32_datadir}/osinfo/LICENSE
+%{mingw32_datadir}/osinfo/VERSION
+%{mingw32_datadir}/osinfo/datamap
+%{mingw32_datadir}/osinfo/device
+%{mingw32_datadir}/osinfo/os
+%{mingw32_datadir}/osinfo/platform
+%{mingw32_datadir}/osinfo/install-script
+%{mingw32_datadir}/osinfo/schema
+
+%files -n mingw64-osinfo-db
+%dir %{mingw64_datadir}/osinfo
+%doc %{mingw64_datadir}/osinfo/LICENSE
+%{mingw64_datadir}/osinfo/VERSION
+%{mingw64_datadir}/osinfo/datamap
+%{mingw64_datadir}/osinfo/device
+%{mingw64_datadir}/osinfo/os
+%{mingw64_datadir}/osinfo/platform
+%{mingw64_datadir}/osinfo/install-script
+%{mingw64_datadir}/osinfo/schema
+%endif
+
 %changelog
-* Wed Dec 28 2022 Muhammad Falak <mwani@microsoft.com> - 20221130-2
```

Comment:
Changelog is also garbled. We need to have only one entry that talks about the upgrade which you are doing.